apps/client/src/app/core/scheme/categories/categories.component.ts

```diff
@@ -0,0 +1,181 @@
+import { Component, Inject, Input } from '@angular/core'
+import { ButtonModule } from 'primeng/button'
+import { DialogModule } from 'primeng/dialog'
+import { FormsModule, ReactiveFormsModule } from '@angular/forms'
+import { ValidatedFormGroup } from '../../../common/validated-form-group/validated-form-group'
+import * as Yup from 'yup'
+import {
+  CategoryService,
+  CategoryDTO as CategoryDTO
+} from '../../../services/category.service'
+import { Toast } from '../../../common/toast/toast.component'
+import { InputTextModule } from 'primeng/inputtext'
+import { InputTextareaModule } from 'primeng/inputtextarea'
+import { Category } from '../../../../shared/types/category.type'
+import { ConfirmationService } from 'primeng/api'
+import { PanelModule } from 'primeng/panel'
+
+@Component({
+  selector: 'categories',
+  standalone: true,
+  imports: [
+    DialogModule,
+    FormsModule,
+    ReactiveFormsModule,
+    ButtonModule,
+    InputTextModule,
+    InputTextareaModule,
+    PanelModule
+  ],
+  templateUrl: './categories.component.html'
+})
+export class CategoryComponent extends ValidatedFormGroup<formPayload> {
+  constructor(
+    @Inject(Toast) private toast: Toast,
+    private confirmationService: ConfirmationService,
+    private categoryService: CategoryService
+  ) {
+    const initialControlValues = {
+      name: '',
+      helpText: ''
+    }
+    const validationSchema = Yup.object({
+      name: Yup.string()
+        .required('El nombre es requerido')
+        .max(50, 'El nombre no puede superar los 50 caracteres'),
+      helpText: Yup.string().required('La ayuda es requerida')
+    })
+
+    super(initialControlValues, validationSchema)
+  }
+
+  errors = {
+    name: '',
+    helpText: ''
+  }
+  @Input() categories: Category[] = []
+  @Input() indicatorIndex: number = 0
+  visibleCreate = false
+  visibleEdit = false
+  fetchCreateCategory: boolean = false
+  fetchEditCategory: boolean = false
+  categoryEdit: formPayloadEdit = {
+    name: '',
+    helpText: '',
+    id: 0
+  }
+  closeDialog() {
+    this.visibleCreate = false
+    this.visibleEdit = false
+    this.formGroup.reset()
+    this.errors = {
+      name: '',
+      helpText: ''
+    }
```

Comment:
Reiniciar el this.errors debería ser parte del método reset

Reply:
como así?